gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/modes/modes/ApplyMode.java

```diff
@@ -31,6 +32,11 @@ public CommonOptions configureAndReturnCommonOptions(ModeOptionContext context)
             public PatchChecksOption patchChecks() {
                 return PatchChecksOption.someChecks(() -> checksToApplySuggestedPatchesFor(context));
             }
+
+            @Override
+            public Map<String, String> extraErrorProneCheckOptions() {
+                return Map.of("SuppressibleErrorProne:Mode", "Apply");
+            }
```

Comment:
To maintain parity with RemoveUnusedMode. Also, potentially a way to move mode interference logic into VisitorStateModifications in the future (see https://github.com/palantir/suppressible-error-prone/issues/180)